network/network_windows.go

```diff
@@ -304,6 +304,7 @@ func (nm *networkManager) configureHcnNetwork(nwInfo *EndpointInfo, extIf *exter
 
 	// Populate subnets.
 	for _, subnet := range nwInfo.Subnets {
+		logger.Info("NextHop is", zap.String("NextHop", subnet.Gateway.String()))
```

Comment:
remove log.. we log when we receive from cns right?

Reply:
logs are removed